sky/packages/sky/lib/src/rendering/binding.dart

```diff
@@ -172,8 +172,12 @@ class FlutterBinding extends HitTestTarget {
       listener(size);
   }
 
+  void _handlePersistentFrameCallback(Duration timeStamp) {
+    beginFrame();
+  }
+
   /// Pump the rendering pipeline to generate a frame for the given time stamp
-  void beginFrame(Duration timeStamp) {
+  void beginFrame() {
```

Comment:
LGTM, but come to think of it, how do we ensure that the animation persistent frame callbacks are called before this one?